src/main/java/net/consensys/orion/http/handler/privacy/PrivacyGroups.java

```diff
@@ -12,12 +12,16 @@
  */
 package net.consensys.orion.http.handler.privacy;
 
+import net.consensys.orion.enclave.PrivacyGroupPayload;
+
 import java.io.Serializable;
 
 public class PrivacyGroups implements Serializable {
```

Comment:
Should "PrivacyGroups" this be singular since this appears to represent a single group?